### ISSUE TITLE
Change event creation settings to offer single events and Dauerveranstaltungen

### ISSUE THREAD

**Issue by osmers** [digitalfabrik/integreat-cms#3070]
### Motivation
Municipalities are complaining about the restriction of 28 days on events bcs some events (exhibitions, Interkulturelle Woche) are longer than that. We created the restriction to make the event calendar clearer (long events are always shown first and put one day events to the back of the list).

### Proposed Solution
Divide the date and time section of event creation into two
One for one-time events and one for Dauerveranstaltungen
Standard selection will be the one-time-event.
It'll have only one date field (no start and end date, bcs they are on the same day)
Reccuring events will stay in this tab.
Events longer than 28 days imported with an external calendar are imported as long term event

Dauerveranstaltungen will be in the second tab. They will have an explanation above what can be considered Dauerveranstaltung and an explanation why it sometimes makes more sense to create recurring events instead).

Make sure to be backwards compatible for the apps (and perhaps consult the app team on the new api design).

### Alternatives
Keep restrictions or remove them completely. 
Have some kind of Freigabeprozess for long events.

### User Story
As a CMS user I want to have the autonomy to create the events that I need in order to reflect what we offer in our municipality. 

### Additional Context
App issue: https://github.com/digitalfabrik/integreat-app/issues/2978
Please notify the app team once this is done.

![image](https://github.com/user-attachments/assets/8140437d-be47-46e6-8d55-ddd57f6981b7)

### Design Requirements
@hauf-toni can you design this according to what we discussed during the conference?
The design can be found here: https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2204-1049&t=tTzbVsG1piOxI0cl-1

**Comment by nikolahoff** [comment]
@osmers question regarding: "Dauerveranstaltungen will be in the second tab." Do you mean that the tile will have 2 tabs?  Something like this?
<img width="524" alt="Bildschirmfoto 2024-10-07 um 12 00 20" src="https://github.com/user-attachments/assets/94afbca9-7e2b-4928-a007-dd62f8b1e5f3">

**Comment by osmers** [comment]
> @osmers question regarding: "Dauerveranstaltungen will be in the second tab." Do you mean that the tile will have 2 tabs? Something like this? <img alt="Bildschirmfoto 2024-10-07 um 12 00 20" width="524" src="https://private-user-images.githubusercontent.com/97434837/374122948-94afbca9-7e2b-4928-a007-dd62f8b1e5f3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyOTc0MDAsIm5iZiI6MTcyODI5NzEwMCwicGF0aCI6Ii85NzQzNDgzNy8zNzQxMjI5NDgtOTRhZmJjYTktN2UyYi00OTI4LWEwMDctZGQ2MmY4YjFlNWYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA3VDEwMzE0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg5MDg4MWU3NDRmOGJkZmU2ZDcyNWYwZGM3Mzk0YjUzZGZmNGMxZmNhNjk4ODEyYmY5ZjI1YjUxMmMwNDJhYjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.MmHPqdjQpcvGTWEDYm9ns5q5rIRXzzl8uSyRvMxX8AM">

@nikolahoff 
Yeah, something like that - I think visually I would have both tabs covering the complete width but that's up to you :) and maybe keep the color the blue we use?

**Comment by nikolahoff** [comment]
It was just a draft to clarify my question, not the visual design yet:) @osmers

**Comment by nikolahoff** [comment]
@osmers @JoeyStk the updated design as discussed is [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2281-108&t=52qtWegVhbUUndFW-1)

**Comment by nikolahoff** [comment]
The adapted design is [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2204-1049&t=tTzbVsG1piOxI0cl-1)

**Comment by osmers** [comment]
@nikolahoff the design is ready, right? Then I will add screenshots to the description
@MizukiTemma or @JoeyStk can either of you give an estimate on how much effort this ticket will be based on the design?

**Comment by JoeyStk** [comment]
I'd say at least medium. There are a few things down the line that should be considered. This is only for the initial work, and doesn't estimate the time for review and work on the suggestions. I'm open for other opinions too :)

**Comment by osmers** [comment]
I'll present this design at the Dialogforum today as something planned for 2025, which I think is realistic, bcs we are not saying when it will be ready. And since we have it planned for Q4 anyways, I assume it will hopefully be ready in Q1 or early/mid Q2 :) any objections?

**Comment by juliankehne** [comment]
Added "Events longer than 28 days imported with an external calendar are imported as long term event" to description

**Comment by JoeyStk** [comment]
@osmers Can this be moved to 2025Q1? :)
